### PR TITLE
Add color option to marker options

### PR DESCRIPTION
**Pull request by arroui**
The color parameter has been added to the MarkerOptions in both the marker.component.ts file and map.service.ts file. This addition enables markers to be customized with different colors, enhancing visual distinction and user experience.

### CODE REVIEW

**Comment by HarelM** [comment]
Thanks!! Can you add this to the docs as well?

**Comment by arroui** [comment]
> Thanks!! Can you add this to the docs as well?

Is the file API.md the only place to add the changes in ? Or have I missed another one ?

**Comment by HarelM** [comment]
Only API.md I believe, oh, and add a line in the change log.

**Comment by arroui** [comment]
I was wondering if I should make a new release?

**Comment by HarelM** [comment]
Sure, why not... A patch version increment...
I'll see if I can do that.

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,7 +4,8 @@ All notable changes to this project will be documented in this file. See [standa
 
 ### [17.4.0](https://github.com/maplibre/ngx-maplibre-gl/compare/v17.1.0...v17.4.0) (2024-02-01)
 
-- Upgrade maplibre-gl to version 4 ([#139]https://github.com/maplibre/ngx-maplibre-gl/pull/139)
+- Add color property to marker [#141](https://github.com/maplibre/ngx-maplibre-gl/pull/141)
```
Comment (HarelM) [inline CHANGELOG.md:7]: Can you move this above the 17.4 version header?

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "ngx-maplibre-gl-srcs",
-  "version": "17.4.0",
+  "version": "17.4.1",
```
Comment (HarelM) [inline package.json:3]: There is an action that will take care of it, I'll run it right after I merge this.

Comment (HarelM) [reply package.json:3]:
```suggestion
  "version": "17.4.0",
```

**Review by HarelM — APPROVED** [review]

**Pull request merged**